test/e2e/volumes.go

```diff
@@ -76,7 +75,6 @@ func DeleteCinderVolume(name string) error {
 	return err
 }
 
-
 // These tests need privileged containers, which are disabled by default.  Run
 // the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
 var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
```

Comment:
Ok, I remember reading that. Still there is only one occurrence of "Feature:Volumes" in all _test/*_, so it's either not being used (much), or there are no e2e's testing volumes that have non-default requirements. For your tests what are the non-default requirements?